AdaptorsRDFSchema/src/au/gov/asd/tac/constellation/graph/schema/rdf/RDFSchemaFactory.java

```diff
@@ -184,69 +200,98 @@ public void completeVertex(final GraphWriteMethods graph, final int vertexId) {
                     graph.setObjectValue(vertexTypeAttribute, vertexId, type);
                 }
             }
+
+            final String label = graph.getStringValue(vertexLabelAttribute, vertexId)!=null? graph.getStringValue(vertexLabelAttribute, vertexId).split("<")[0]:"";
```

Comment:
@antares1470 I was getting a null pointer error here so I did a null check. However when I load `constellation.ttl` for example nodes are connected to Unknown nodes, probably because the label is empty?